store/postgres/src/deployment_store.rs

```diff
@@ -1043,17 +1050,17 @@ impl DeploymentStore {
     ) -> Result<Option<Entity>, StoreError> {
         let conn = self.get_conn()?;
         let layout = self.layout(&conn, site)?;
-        layout.find(&conn, &key.entity_type, &key.entity_id, block)
+        layout.find(&conn, &key, block)
     }
 
     /// Retrieve all the entities matching `ids_for_type` from the
```

Comment:
Seems to have some more details we could add here now

Reply:
Improved the comment a bit.